nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf

```diff
@@ -91,5 +93,13 @@ process GET_SOFTWARE_VERSIONS {
         yaml.dump(versions, f, default_flow_style=False)
     with open("software_versions_mqc.yml", 'w') as f:
         yaml.dump(versions_mqc, f, default_flow_style=False)
+
+    yaml_version = {}
+    yaml_version["${getProcessName(task.process)}"] = {
```

Comment:
This is probably just to make the pytests happy .. but if we have it anyway, should we also include it into the yaml containing the collected versions?

Reply:
Good idea 👍🏽

Reply:
Added in https://github.com/nf-core/modules/pull/764 and will re-install here once that has been merged.